src/cleanup_queue-inl.h

```diff
@@ -0,0 +1,60 @@
+#ifndef SRC_CLEANUP_QUEUE_INL_H_
+#define SRC_CLEANUP_QUEUE_INL_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "base_object.h"
+#include "cleanup_queue.h"
+#include "memory_tracker-inl.h"
+#include "util.h"
+
+namespace node {
+
+inline void CleanupQueue::MemoryInfo(MemoryTracker* tracker) const {
+  ForEachBaseObject([&](BaseObject* obj) {
+    if (obj->IsDoneInitializing()) tracker->Track(obj);
+  });
+}
+
+inline size_t CleanupQueue::SelfSize() const {
+  return sizeof(CleanupQueue) +
```

Comment:
Actually, `Environment::SelfSize()` should subtract `sizeof(cleanup_queue_)` now since that's stack-allocated..

Reply:
Thanks for spotting this. Fixed.